linera-service/src/project.rs

```diff
@@ -230,6 +242,26 @@ impl Project {
         (linera_sdk_dep, linera_sdk_dev_dep)
     }
 
+    /// Converts a string to PascalCase.
+    fn to_pascal_case(s: &str) -> String {
+        let mut result = String::new();
+        // Start a word with capital letter.
+        let mut capitalize = true;
+        for c in s.chars() {
+            if c == '-' {
+                // New word incoming.
+                capitalize = true;
+            } else if capitalize {
+                result.push(c.to_ascii_uppercase());
+                // If we capitalize a letter, we should not capitalize the next one.
+                capitalize = false;
+            } else {
+                result.push(c);
+            }
+        }
+        result
+    }
+
```

Comment:
We already have the [https://docs.rs/convert_case/latest/convert_case/](convert_case) crate as a dependency — maybe we should use that?

Reply:
Done in [6e150d1](https://github.com/linera-io/linera-protocol/pull/2578/commits/6e150d1648d4ce6f020e7cf77c3d65baf1f43b4e)